src/BuildScriptGeneratorCli/BuildScriptGeneratorCli.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RuntimeFrameworkVersion>2.1</RuntimeFrameworkVersion>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Could you create a 'new' project in VS or dotnet cli and copy paste most of the content from there to these .csproj files...and also we have package references to 2.2 which need to be moved to 3.1 as well

Reply:
yepp .. i will do the clean up that includes the corresponding package references in dotnetcore 3.1, wanted to quickly test my theory that moving to 3.1 will work .. so i will do the changes now .. only 1 test failed .. but it will work I guess